exercises/book-store/src/example/java/BookStore.java

```diff
@@ -0,0 +1,76 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class BookStore{
+
+	private int bookPrice, maxGroupSize;
+	private double discountIncrement;
+	private List<Integer> books;	
+
+	public BookStore (List<Integer> books){
+		this.books = books;
+		this.bookPrice = 8;
+	}	
+
+	public double CalculateTotalCost(){
```

Comment:
Method names should be `camelCase`, but not `CapitalWords` in Java.  Save the `CapitalWords` for class names.